doc/man3/EVP_PKEY_copy_parameters.pod

```diff
@@ -55,7 +58,8 @@ doesn't use parameters.
 These functions EVP_PKEY_copy_parameters() returns 1 for success and 0 for
 failure.
 
-The function EVP_PKEY_parameters_eq() and EVP_PKEY_eq() return 1 if their
+The functions EVP_PKEY_cmp_parameters(), EVP_PKEY_parameters_eq(), 
+EVP_PKEY_cmp() and EVP_PKEY_eq() return 1 if their
```

Comment:
Please drop this hunk. We should make the _eq as the preferred name.

Reply:
I have added a new commit adjusting the documentation. Please confirm this is in line with your suggested update.

Reply:
Perfect now.